docs/database/mongodb.mdx

```diff
@@ -40,6 +40,7 @@ export default buildConfig({
 | `transactionOptions`       | An object with configuration properties used in [transactions](https://www.mongodb.com/docs/manual/core/transactions/) or `false` which will disable the use of transactions.                                                                                                                                |
 | `collation`                | Enable language-specific string comparison with customizable options. Available on MongoDB 3.4+. Defaults locale to "en". Example: `{ strength: 3 }`. For a full list of collation options and their definitions, see the [MongoDB documentation](https://www.mongodb.com/docs/manual/reference/collation/). |
 | `allowAdditionalKeys`      | By default, Payload strips all additional keys from MongoDB data that don't exist in the Payload schema. If you have some data that you want to include to the result but it doesn't exist in Payload, you can set this to `true`. Be careful as Payload access control _won't_ work for this data.          |
+| `acceptIDOnCreate`         | Enable this flag if you want to thread your own ID to create operation data without using a custom ID field.                                                                                                                                                                                                 |
```

Comment:
I would name this `allowIDOnCreate`. We use `allow` as the verb in a few places instead of `accept`.
```md
| `allowIDOnCreate` | Set to `true` to use the `id` passed in data on the create API operations without using a custom ID field.
```

Reply:
to throw my little spanner in there.. 

if using kebab would it be `allow-i-d-on-create` or `allow-id-on-create`?

so shouldn't it be `allowIdOnCreate`?

(uppercase acronyms in camel case break my brain) 😄

Reply:
> to throw my little spanner in there.. 
> 
> if using kebab would it be `allow-i-d-on-create` or `allow-id-on-create`?
> 
> so shouldn't it be `allowIdOnCreate`?
> 
> (uppercase acronyms in camel case break my brain) 😄 

This is always a trap. We prefer to capitalize acronyms, for example: payload.findByID. It is more common in JS in general like `date.toISOString()`.